invenio_app_ils/ui/src/invenio_app_ils/pages/backoffice/DocumentDetails/components/DocumentPendingLoans/DocumentPendingLoans.js

```diff
@@ -43,7 +43,18 @@ export default class DocumentPendingLoans extends Component {
   };
 
   prepareData(data) {
-    return data.hits.map(row => formatter.loan.toTable(row));
+    return data.hits.map(row => {
+      const serialized = formatter.loan.toTable(row);
+      delete serialized['Created'];
```

Comment:
For these multiple deletions, lodash offers `_.pick` and `_.omit` for operations like this.

https://lodash.com/docs/#pick
https://lodash.com/docs/#omit

If you decide to use it, there are several occurrences of multiple properties deletion further down.

Reply:
❤️